Cpp-Fizzbuzz/test.cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+#include "fizzbuzz.hpp"
+#include "json.hpp"
+
+using json = nlohmann::json;
+
+std::string fizzbuzz_ref(int n) {
+  if (n % 15 == 0) {
+    return "FizzBuzz";
+  } else if (n % 5 == 0) {
+    return "Buzz";
+  } else if (n % 3 == 0) {
+    return "Fizz";
+  } else {
+    return std::to_string(n);
+  }
+}
+
+int main() {
+  json result;
+  std::srand(std::time(nullptr));
+
+  int test_int_1 = 30;
+  if (fizzbuzz(test_int_1) == fizzbuzz_ref(test_int_1)) {
```

Comment:
Small nit but I think it would be good to include a static assertion here to make sure that fizzbuzz returns the same type as fizzbuzz_ref and takes in the same type. 

Or else the equality sign would not compute.